_data/students.yml

```diff
@@ -184,3 +184,9 @@
   name: Rasagnath Reddy
```

Comment:
By the way, I found there is one student who forgot to add a dash (`-`).

Reply:
Create and issue and make PR